examples/ct/structures_test.go

```diff
@@ -43,4 +58,54 @@ func TestGetCTLogIDNotLoaded(t *testing.T) {
 }
 
 func TestSignV1SCTForCertificate(t *testing.T) {
+	cert, err := fixchain.CertificateFromPEM(testonly.LeafSignedByFakeIntermediateCertPem)
+
+	if err != nil {
+		t.Fatalf("failed to set up test cert: %v", err)
+	}
+
+	km := setupMockKeyManager()
+
+	got, err := SignV1SCTForCertificate(km, cert, fixedTime)
+
+	if err != nil {
+		t.Fatalf("create sct for cert failed", err)
+	}
+
+	logID, err := base64.StdEncoding.DecodeString(ctMockLogID)
+
+	if err != nil {
+		t.Fatalf("failed to decode test log id: %s", ctMockLogID)
+	}
+
+	var idArray [sha256.Size]byte
+	copy(idArray[:], logID)
+
+	expected := ct.SignedCertificateTimestamp{SCTVersion: 0,
+		LogID:      ct.SHA256Hash(idArray),
+		Timestamp:  1504786523000000,
+		Extensions: ct.CTExtensions{},
+		Signature: ct.DigitallySigned{HashAlgorithm: ct.SHA256,
+			SignatureAlgorithm: ct.RSA,
```

Comment:
Yes but it doesn't seem to change / fix all indentation problems. Changed layout slightly and I think it's right now.